.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+This issue template is meant mainly for bug reports and feature requests.
+
+You can still use an issue to ask for help. In this case
+you should remove the template contents or use it as a guideline
+for providing information for debugging.
+-->
+
+### Kind of issue
+
+> Uncomment only one of these:
+> 
+> Bug
+> Feature request
+
+### Observed behavior
+
+
+### Expected/desired behavior
+
+
+### Details on how to reproduce (minimal and precise)
+
+
+### Information about the environment:
+
+- Glusterd2 version used (e.g. v4.1.0 or master): 
```

Comment:
The users won't know how to get this information - they will have to go look elsewhere. It's more useful to ask users to run specific commands to get the information.

For example, ask users to provide: output of `glusterd2 --version` and `etcd --version`

Reply:
done add one more section for it